siphon/simplewebservice/wyoming.py

```diff
@@ -36,6 +36,12 @@ def request_data(cls, time, site_id, **kwargs):
             The three letter ICAO identifier of the station for which data should be
             downloaded.
 
+        region (optional) : str
+            Region for the sounding data. Valid regions are `North America`,
+            `South America`, `South Pacific`, `New Zealand`, `Antarctica`,
+            `Arctic`, `Europe`, `Africa`, `Southeast Asia`, `Mideast` Defaults
```

Comment:
Period after `Mideast`. Also, should probably replace backticks with single quotes...I think.